source/base/data_out_base.cc

```diff
@@ -6220,7 +6220,8 @@ write_pvd_record (std::ostream &out,
                   const std::vector<std::pair<double,std::string> >  &times_and_names) const
 {
   AssertThrow (out, ExcIO());
-
+   
+  out.precision(12);
```

Comment:
Fair enough, I'm happy with that if we're going to consistently use it from this point forward :-)